modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/js/utils/EventListener.js

```diff
@@ -12,21 +12,18 @@
  * details.
  */
 
-(function () {
-	AUI().applyConfig({
-		groups: {
-			blogs: {
-				base: MODULE_PATH + '/blogs/js/',
-				combine: Liferay.AUI.getCombine(),
-				filter: Liferay.AUI.getFilterConfig(),
-				modules: {
-					'liferay-blogs': {
-						path: 'blogs.js',
-						requires: ['aui-base', 'liferay-form'],
-					},
-				},
-				root: MODULE_PATH + '/blogs/js/',
-			},
-		},
-	});
-})();
+export default class EventListener {
```

Comment:
Do you really need a custom class for this? You could just as well handle this state internally in your `Blogs` class and avoid the additional class.